script.js

```diff
@@ -0,0 +1,89 @@
+
+var tiles = document.querySelectorAll(".tile")
+
+var moves = 0
+
+
+function tilesClicked(evt) {
+	// console.log(evt);
+	var tile = evt.target
+	console.log(evt)
+	if(tile.innerHTML === '') {
+		moves = moves + 1;
+		if(moves % 2 === 0) {
+			tile.innerHTML = "o";
+			tile.className += " odd";
+		}
+		else {
+			tile.innerHTML = "x";
+			tile.className += " even";
+		}
+	}
+}
+
+for (var i = 0; i < tiles.length; i++) {
+	tiles[i].addEventListener('click', tilesClicked);
+}
+
+document.getElementById('button').addEventListener('click', function () {
```

Comment:
On larger projects, you'll want to make sure to be very descriptive with the names of classes/ids. For example, instead of just `button`, the id could be `reset-button`